doc-src/SASS_REFERENCE.md

```diff
@@ -2499,6 +2499,11 @@ The same mixins can be done in the `.sass` shorthand syntax:
 
 **Note:** when the `@content` directive is specified more than once or in a loop, the style block will be duplicated with each invocation.
 
+Some mixins may require a passed content block or may have different
+behavior depending on whether a content block was passed. The
+`content-exists()` function will return true when a content block is
+passed to the current mixin and can be used to implement such behaviors.
```

Comment:
Typically, we don't list functions in the reference unless they're crucial to the use of a broader feature (like `map-get()`). When we do list them, though, we should link to their documentation.